blueprints/acceptance-test/index.js

```diff
@@ -17,10 +17,6 @@ module.exports = useTestFrameworkDetector({
       testFolderRoot = pathUtil.getRelativeParentPath(options.entity.name, -1, false);
     }
 
-    let destroyAppExists = fs.existsSync(
```

Comment:
Hmm, I think this still needs to exist no? This isn't about the ember-cli version that is being used by ember-source, but rather about the current project layout _when the blueprint is ran_...

Reply:
You're right, thanks for pointing this out. I enjoy deleting code so much, I just jumped right in 🙈